ovos_core/transformers.py

```diff
@@ -0,0 +1,124 @@
+from typing import Optional, List
+
+from ovos_plugin_manager.metadata_transformers import find_metadata_transformer_plugins
+from ovos_plugin_manager.templates.transformers import UtteranceTransformer
+from ovos_plugin_manager.text_transformers import find_utterance_transformer_plugins
+
+from ovos_utils.json_helper import merge_dict
+from ovos_utils.log import LOG
+
+
+class UtteranceTransformersService:
```

Comment:
This can be imported from `neon_transformers.text_transformers`. The only difference I see here is the `plugins` property here is called `modules` there and the default value for `self.config` is different.

Reply:
dont want to drag the dependency, and i expect these might diverge over time to include neon specific stuff, want core to be quite standalone and stable. In this case the plugin classes are what matters, how they are consumed is application specific most of the time

main concern for neon being our downstream would be us pinning an old version and then blocking neon from updating core because of that